modules/nextMillenniumBidAdapter.js

```diff
@@ -39,7 +62,19 @@ export const spec = {
             'scrollTop': window.pageYOffset || document.documentElement.scrollTop
           }
         },
-        ...bid.ortb2
+        ...bid.ortb2,
+        'imp': [{
+          'banner': {
+            'format': (bid.sizes || []).map(s => { return {w: s[0], h: s[1]} })
+          },
+          'ext': {
+            'prebid': {
+              'storedrequest': {
+                'id': id
```

Comment:
@bretg here is another use case of an adapter explicitly referencing stored req id

Reply:
They're good - the storedrequest id comes from a bidder-specific parameter. The issue that had been discussed is when the publisher defines a storedrequest for their main PBS provider. That data is passed to bidders, that may be running their own version of PBS for which that id would mean nothing.